cmd/statsdaemon/main.go

```diff
@@ -20,7 +20,7 @@ import (
 	"net/http"
 	_ "net/http/pprof"
 
-	"github.com/stvp/go-toml-config"
+	"github.com/rakyll/globalconf"
```

Comment:
this library is not maintained anymore.
we use https://github.com/grafana/globalconf now